controllers/articles-controllers.js

```diff
@@ -0,0 +1,10 @@
+const { fetchArticle } = require('../models/articles-models')
+
+exports.getArticle = (req, res, next) => {
+    const {article_id} = req.params
+    fetchArticle(article_id)
+    .then((article) => {
+        res.status(200).send({'article': article})
```

Comment:
Well done sending it back on a key 👍 Remember we can achieve the same result by just sending `{article}` though